pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -76,23 +92,28 @@ func (r *Client) CheckSnapshotReady(vmRef ref.Ref, snapshot string) (ready bool,
 		err = liberr.Wrap(err)
 		return
 	}
-	jobs, err := r.getJobs(correlationID)
+	events, err := r.getEvents(correlationID)
 	if err != nil {
 		err = liberr.Wrap(err)
 		return
 	}
-	if len(jobs) < 1 {
-		err = liberr.New("No jobs found for correlation ID", "correlationID", correlationID)
+	if len(events) < 1 {
+		err = liberr.New("No event found for correlation ID", "correlationID", correlationID)
 		return
 	}
-	ready = true
-	for _, job := range jobs {
-		switch job.MustStatus() {
-		case ovirtsdk.JOBSTATUS_FAILED, ovirtsdk.JOBSTATUS_ABORTED:
+	for _, event := range events {
+		code, exists := event.Code()
+		if !exists {
+			err = liberr.New("The event does not have a code.", "event", event)
```

Comment:
sorry about not writing about that before but I just noticed that `err` is set in this case - and an invalid event shouldn't lead to a failure, let me post a change for this